showcase/src/assets/translations/fr.json

```diff
@@ -267,20 +267,22 @@
-      "DOCUMENTATION_MAIN_DESC": "Vous voulez plus d'infos sur Stark? Consultez nos liens de documentation ci-dessous!",
-      "DOCUMENTATION_CORE_DESC": "<p>Stark Core, aussi appelé stark-core, est un module fournissant des APIs réutilisables, comme par exemple une API de routing, de logging, de log shipping,etc... </p><p>Vous trouverez la documentation la plus récente en cliquant <a href=\"https://stark.nbb.be/api-docs/stark-core/latest\" target=\"_blank\" rel=\"noopener noreferrer\">ici</a>.</p><p>Ou allez jeter un oeil aux anciennes versions de l'API par <a href=\"https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-core \" target=\"_blank\" rel=\"noopener noreferrer\">ici</a>.</p>",
-      "DOCUMENTATION_UI_DESC": "<p>Stark UI fournit des composants UI réutilisables: table de données, boite de dialogue, ... mais également des thèmes.</p><p>Pour consulter la dernière documentation de cette API, cliquez <a href=\"https://stark.nbb.be/api-docs/stark-ui/latest\" target=\"_blank\" rel=\"noopener noreferrer\">ici</a>.</p><p> Ou allez jeter un oeil aux anciennes versions de l'API par <a href=\"https://github.com/NationalBankBelgium/stark/tree/gh-pages/api-docs/stark-ui \" target=\"_blank\" rel=\"noopener noreferrer\">ici</a>.</p>",
-      "DOCUMENTATION_SHOWCASE_DESC": "<p>Le Showcase, vous vous y trouvez en ce moment même! </p><p>Amusez-vous à parcourir notre liste de composants visuels ainsi que leur documentation et à découvrir de vos propres yeux nos guidelines en matière de design. </p>Si vous le souhaitez, vous pouvez toujours consulter l'ancienne version de ce showcase en cliquant sur ce <a href=\"https://github.com/NationalBankBelgium/stark/tree/gh-pages/showcase\" target=\"_blank\" rel=\"noopener noreferrer\">lien</a>.</p>",
-      "TITLE": "Home"
+      "CORE":"Stark Core",
+      "DESCRIPTION_DETAIL":"<p>Stark fournit les principaux outils pour accélérer le développement front-end.<br>Vous y trouverez un build fiable et réutilisable basé sur Webpack, un projet Starter, des modules core et ui fournissant des composants ui incroyables, ... <br>Tous ces modules sont comme des LEGO: ajoutez ce dont vous avez besoin, ni plus, ni moins!</p>",
+      "DOCUMENTATION_CORE_DESC": "<p>Stark Core, aussi appelé stark-core, est un module fournissant des APIs réutilisables, comme par exemple une API de routing, de logging, de log shipping,etc... </p>",
+      "DOCUMENTATION_MAIN_TITLE": "Documentation",
+      "DOCUMENTATION_SHOWCASE_DESC": "<p>Le Showcase, vous vous y trouvez en ce moment même! <br> Amusez-vous à parcourir notre liste de composants visuels ainsi que leur documentation et à découvrir de vos propres yeux nos guidelines en matière de design.</p>",
+      "DOCUMENTATION_UI_DESC": "<p>Stark UI fournit des composants UI réutilisables: table de données, boite de dialogue, ... mais également des thèmes.</p>",
+      "GETTING_STARTED":"Démarrer",
+      "INTRODUCTION": "<p>Un framework front-end basé sur Angular 7</p>",
```

Comment:
The `<p>` in this translation is unnecesary because there is already a `<p>` where this translation is used... could you remove it from here?